app/src/main/res/values/strings.xml

```diff
@@ -289,4 +289,19 @@
   <string name="wikidata_edit_failure">Failed to update corresponding Wikidata entity!</string>
   <string name="menu_set_wallpaper">Set wallpaper</string>
   <string name="wallpaper_set_successfully">Wallpaper set successfully!</string>
+  <string name="quiz">Quiz</string>
+  <string name="quiz_question_string">Is it Ok to Upload ? </string>
+  <string name="question">Question</string>
+  <string name="result">Result</string>
+  <string name="quiz_back_button">Do you want to end Quiz ? </string>
+  <string name="quiz_alert_message">Percentage of images reverted is more than 50%.Take Quiz to get familiar with the topicness and copyrights assosciated with creative commoms</string>
+  <string name="selfie_answer">Selfies do not have much encyclopedic value. Please do not upload a picture of your self unless you are famous enough and already have a Wikipedia article about you.</string>
```

Comment:
"...of yourself..."

Reply:
I would just write "unless there is already a Wikipedia article about you." because some people might stop at "famous enough" and think "oh yeah I am famous enough" and stop reading.